docs/man/man1/kubectl-annotate.1

```diff
@@ -1,4 +1,4 @@
-.TH "KUBERNETES" "1" " kubernetes User Manuals" "Eric Paris" "Jan 2015"  ""
+.TH "KUBERNETES" "1" ""  ""
```

Comment:
The date shows up at the end, I'll look into making both of those things possible.